Dockerfile

```diff
@@ -29,10 +29,16 @@ USER root
 
 RUN set -x && \
     # Install additional required tools \
-    apk add --no-cache clamav-clamdscan coreutils gawk && \
+	apt-get update && \
+	apt-get install -y --no-install-recommends \
+		clamdscan \
+		coreutils \
+		gawk && \
```

Comment:
This should not be necessary anymore with the non-alpine image (same for the symlinking below)